src/components/SuggestionsList/SuggestionsList.css

```diff
@@ -1,6 +1,10 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 @import "src/css/variables/colors.css";
 
+.y-suggestions-list.ms-Callout {
+  animation-name: none;
+}
```

Comment:
@adrianmg here's a chance to change the box-shadow/border-radious stuff. The default looks like it matches Zeplin but I think you wanted to change it?

current settings:-
```cs
border-radius: 1px;
box-shadow: 0 0 12px 0 rgba(0, 0, 0, 0.2);
```

Reply:
The border-radius might be applied to the container but is probably not being displayed properly due to the border-radius value of the first and last row.

Attaching the image showing this…

![image](https://user-images.githubusercontent.com/589285/35821042-8c4261b6-0a5c-11e8-9157-be6d68d5a7f5.png)

Reply:
Left is wrong, right is correct

![image](https://user-images.githubusercontent.com/589285/35821139-d5c01b08-0a5c-11e8-9b99-8e2bafa33900.png)

Reply:
ah ok, taking a look